Makefile

```
@@ -0,0 +1,14 @@
+all:	test build
+.PHONY: all
+
+build:
+	CGO_ENABLED=0 go build -o dist/tcplb ./cmd/tcplb/...
```

nit:

```suggestion
	CGO_ENABLED=0 go build -o dist/tcplb ./cmd/tcplb
```

The `/...` means "package and children", but when we "go build" a binary we should pick only `package main`. I'm not sure this work work if ./cmd/tcplb/ had other packages nested in it (which is a bit weird if you think about it, but it happens).